src/components/authentication/login.jsx

```diff
@@ -1,22 +1,130 @@
 import React from 'react';
 import styled from 'styled-components';
 
-const StyledButtonExample = styled.button`
+import { Link } from 'react-router-dom';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Image from 'react-bootstrap/Image';
+import Form from 'react-bootstrap/Form';
+
+import logoImage from '../../images/HospiceLogo.png';
+
+const SContainer = styled(Container)`
+  display: flex;
+  text-align:center;
+  justify-content: center;
+  box-sizing: border-box;
+  height: 100vh;
+  padding: 20vh 0;
+
+  @media only screen and (min-width: 800px) {
+    background-color: #ECECEC;
+    padding: 10vh 10vw;
+  }
+`;
+
+const StyledRow = styled(Row)`
+  display: flex;
+  margin:0;
+  padding:0;
+  box-sizing: border-box;
+  text-align: center;
+  justify-content: center;
+`;
+
+const SCol = styled(Col)`
+  background-color: white;
+
+  @media only screen and (min-width: 800px) {
+    padding: 50px 6vw 80px 6vw;
+    border-radius: 6px;
+  }
+
+`;
+
+/* changes width of input fields */
+const Credentials = styled(Form)`
+  width: 100%;
+`;
+
+const SignIn = styled.button`
+  border-radius: 5px;
+  padding: 6px 0px; 
+  width: 48%;
+  font-family: Roboto;
```

Comment:
change font-family to fontFamily to get the style to apply.

Reply:
For me, fontFamily doesn't change the font. Do you still want me to change it?

Reply:
Thats really weird, because the opposite thing happens for me. Maybe you could add them both? I